### PR TITLE
feat: add paralax section

### PR DESCRIPTION
**Pull request by KseniyaSokolovskaya**
add paralax section with perfect pixel

### CODE REVIEW

File: scss/components/_common.scss [hunk 1]
```diff
@@ -111,46 +111,129 @@ body {
...
-
-      &:hover {
-        background-color: $color-btn-hover;
-        border-color: $color-btn-hover;
-      }
+  .btn {
+    width: 151px;
+    text-align: center;
+    font-size: 17px;
+    font-family: 'Raleway',sans-serif;
```
Comment (angelinabylitskaya) [inline scss/components/_common.scss]: Mode fonts to a variable. Consider creating a mixin for fonts definitions.

File: scss/components/_common.scss [hunk 2]
```diff
@@ -111,46 +111,129 @@ body {
...
+    position: absolute;
+    width: 100%;
+    height: 30px;
+    background: linear-gradient(45deg, $color-primary 50%, $color-accent 50%);
+    left: 0;
+    bottom: 0;
+  }
+}
+
+.title{
```
Comment (angelinabylitskaya) [inline scss/components/_common.scss]: Missed space before `{`

File: scss/components/_common.scss [hunk 3]
```diff
@@ -111,46 +111,129 @@ body {
...
 
+.blockquote {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 881px;
+  margin: 0 auto;
+
+  &__text {
+    color: #fafafa;
```
Comment (angelinabylitskaya) [inline scss/components/_common.scss]: Move colors to variables.

File: scss/components/_common.scss [hunk 4]
```diff
@@ -111,46 +111,129 @@ body {
...
+    font-size: 30px;
+    line-height: 40px;
+    word-spacing: 2.1px;
+    font-family: 'Myriad Pro';
```
Comment (angelinabylitskaya) [inline scss/components/_common.scss]: Move font to a variable. Add alternative font.

File: scss/components/_common.scss [hunk 5]
```diff
@@ -111,46 +111,129 @@ body {
...
+    font-weight: normal;
+    padding: 64px 0 45px;
+    position: relative;
+
+    &::before {
+      content: "“";
```
Comment (angelinabylitskaya) [inline scss/components/_common.scss]: Better to use background image or symbol.

**Review by angelinabylitskaya — CHANGES_REQUESTED** [review]
Please, fix code in accordance with comments. All issues can be found several times.

**Comment by angelinabylitskaya** [comment]
image
Design mismatch.

File: scss/components/_common.scss [hunk 6]
```diff
@@ -97,60 +98,144 @@ body {
...
-      border: 1px solid $color-txt-primary;
-
-      &:hover {
-        background-color: $color-btn-hover;
-        border-color: $color-btn-hover;
-      }
+  padding: 34px 0;
+
+  .btn {
+    width: 151px;
```
Comment (kobweb) [inline scss/components/_common.scss:124]: Don't use fixed width, use padding and min-width instead.

File: scss/components/_common.scss [hunk 7]
```diff
@@ -97,60 +98,144 @@ body {
...
+    text-align: center;
+    @include font('Raleway', 600, 17px);
+    color: $color-txt-primary; 
+    user-select: none;
+    text-transform: uppercase;
+    background: transparent; 
+    padding: 14px 0;
+    outline: none;
+    margin-right: 17px;
```
Comment (kobweb) [inline scss/components/_common.scss:133]: Use some kind of modificator for indents, in perspective not all your buttons will have an indent.

**Review by kobweb — APPROVED** [review]

**Pull request merged**